core/src/BarcodeFormat.cpp

```diff
@@ -1,7 +1,7 @@
 /*
-* Copyright 2016 Nu-book Inc.
-* Copyright 2016 ZXing authors
-*/
+ * Copyright 2016 Nu-book Inc.
```

Comment:
Looks awesome, but can you please revert those unrelated "* ..." -> " *..." whitespace changes?

Reply:
OK